packages/utilities/fast-gestures/src/interfaces/pointer-gesture-info.ts

```diff
@@ -0,0 +1,7 @@
+import { PointerTarget } from "../pointer-target";
+import { IPointerInfo } from "./pointer-info";
+
+export interface IPointerGestureInfo {
```

Comment:
@mlijanto can we drop "I" as a prefix to the interfaces across the PR?

Reply:
@chrisdholt I've removed the "I" prefix from most of the interfaces, but there are some cases where removing the prefix would create a conflicting name between the interface and the class that implements it. Could I leave the "I" prefix in these cases, or do you have any guidance or pattern I should follow?

Reply:
Can you point to those scenarios? I see similar names but I don't see any with conflicts - I could be missing it.

Reply:
There's a few of them, such as `IPoint` and `Point`, `IEvent` and `Event`. But this is no longer an issue. Rob and Nick gave a guidance to add the prefix "Default" to the implementations so I could remove the "I" prefix from all the interfaces. Thanks!